test/Cpp/src/ActionWithNoReaction.lf

```diff
@@ -11,7 +11,7 @@ reactor foo {
   logical action a
 
   reaction(x) -> y, a {=
-    y.set(2*(*x.get()));
+    y.set(2*(*x.undefined_name15291838()));
```

Comment:
This looks like an unintended and unrelated change?

Reply:
Yes this is unintended, probably has something to do with the code editor. I'll remove it.